pom.xml

```diff
@@ -67,16 +67,10 @@
     <dependencies>
         <dependency>
             <groupId>edu.illinois.cs</groupId>
-            <artifactId>testrunner-maven-plugin</artifactId>
+            <artifactId>testrunner-core-plugin</artifactId>
             <version>1.1</version>
```

Comment:
Thanks a lot for the changes. Overall, the changes in this PR now looks good. We will have to first update and release testrunner version 1.2 before we can merge this PR.

Please help by updating the version of testrunner to 1.2 in the testrunner PR, update all dependencies of testrunner in iDFlakies to 1.2 as well, and finally update iDFlakies to version 1.2.0 itself (e.g., iDFlakies' pom.xml files, files under pom-modify, and the README). I will look to test your code and review the testrunner PR again soon.

Reply:
Hi Wing, I have a question: currently testrunner's pom.xml specifies the version to be 1.2-SNAPSHOT. Do I need to change it to 1.2 or just leave it as 1.2-SNAPSHOT?

Reply:
The pom for testrunner should be 1.3-SNAPSHOT (SNAPSHOT generally refers to the next, but unreleased, version). The next version of testrunner to be released will be 1.2 but I will set that locally when I do the release, so your changes are just to help prepare for what is needed after my release

Reply:
I notice that you asked earlier whether to add the modify_gradle.sh script to the PR or not. Please do add the script if you have it ready. It will make testing the changes on gradle projects easier.

Reply:
iDFlakies and testrunner are updated accordingly.

As for modify_gradle, @ericssy will be responsible for it.

Reply:
Besides the modify_gradle script, please also provide the README you left as a comment earlier. Specifically, please include that as README-gradle.md. Remember to update the versions there. We will merge this PR once you make the changes and the testrunner version is deployed

Reply:
I created a new PR #32 with the modify_gradle script and a README.